docs/install/1-install-server.rst

```diff
@@ -301,6 +302,10 @@ Initialize the PostgreSQL server:
 
     .. code-block:: sh
 
+        # TODO: the following step was failing because of missing langpacks, I had to install them with
+        # `sudo dnf install langpacks-en glibc-all-langpacks -y`
+        # I'm not sure whether this is specific to my setup of more general
+
         sudo su - postgres -c "postgresql-setup --initdb"
```

Comment:
This step was failing with following error:
`initdb: error: invalid locale settings; check LANG and LC_* environment variables`
Which comes from the fact that the system is using a locale that is not installed:

current status of the system locales settings:
```sh
[rocky@issue-7042 ~]$ localectl status
   System Locale: LANG=en_US.UTF-8     <---- Not installed
       VC Keymap: us
      X11 Layout: us
```

Installed locales:

```sh
[rocky@issue-7042 ~]$ locale -a

locale: Cannot set LC_CTYPE to default locale: No such file or directory
locale: Cannot set LC_MESSAGES to default locale: No such file or directory
locale: Cannot set LC_COLLATE to default locale: No such file or directory
C
C.utf8
POSIX
```


Managed to make the command work by changing to a locale that is installed :
```sh
[rocky@issue-7042 ~]$ sudo localectl set-locale LANG=C.utf8
```

An alternative fix involved installing the missing locale through 
```
[rocky@issue-7042 ~]$ sudo dnf install langpacks-en glibc-all-langpacks -y
```

Only lead I have on why this is happening is that it comes from my ssh configuration that used to have:
```txt
# file : /etc/ssh/ssh_config
#   SendEnv LANG LC_*    <-- this wasn't commented out
```